gravitee-am-repository/gravitee-am-repository-api/src/main/java/io/gravitee/am/repository/management/api/DomainRepository.java

```diff
@@ -32,13 +32,15 @@
  */
 public interface DomainRepository extends CrudRepository<Domain, String> {
 
+    Flowable<Domain> findAllByReferenceId(String environmentId, int page, int size);
+
+    Flowable<Domain> search(String environmentId, String query, int page, int size);
```

Comment:
I think both page and size are not used in the repository layer. We can remove them.